app/src/main/java/de/danoeh/antennapod/fragment/preferences/ImportExportPreferencesFragment.java

```diff
@@ -181,14 +181,11 @@ private void exportDatabase() {
     }
 
     private void importDatabase() {
-        // setup the alert builder
-        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
-        builder.setTitle(R.string.database_import_label);
-        builder.setMessage(R.string.database_import_warning);
-
-        // add a button
-        builder.setNegativeButton(R.string.no, null);
-        builder.setPositiveButton(R.string.confirm_label, (dialog, which) -> {
+        final AlertDialog.Builder alert = new AlertDialog.Builder(getActivity());
+        alert.setTitle(R.string.database_import_label);
+        alert.setMessage(R.string.database_import_warning);
+        alert.setNegativeButton(R.string.no, null);
+        alert.setPositiveButton(R.string.confirm_label, (dialog, which) -> {
```

Comment:
What is the point of renaming here? That makes it harder to use `git blame` without adding real value. Please revert.